CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased Changes
 
+## 0.4.1 (2021-02-03)
+* Implemented `IntoIterator` for `&Arena` and `&mut Arena`.
```

Comment:
These changes aren't released yet. Can you add a bullet point under the "Unreleased Changes" section instead and change the revert the version bump?

When the time is right, I'll create a new release and bump the version with any features at the time.

Reply:
Yes, I was doing wrong XD